documentation/markdown/features.md

```diff
@@ -0,0 +1,90 @@
+# Features
+
+Below is a non-exhaustive listing of the features available to a server instance,
+depending on the chosen configuration.
+The core feature of the CSS is how it uses Components.js to configure its components,
+so any of the features below can always be adapted or replaced with custom components if required.
+See this [tutorial](https://github.com/CommunitySolidServer/tutorials/blob/main/custom-configurations.md)
+and/or this [example repository](https://github.com/CommunitySolidServer/hello-world-component)
+for more information on that.
+
+To generate configurations with some of these features enabled/disabled,
+you can use the [configuration generator](https://communitysolidserver.github.io/configuration-generator/).
+
+## Authentication
+
+Clients are identified based on the contents of DPoP tokens,
+as described in the [Solid-OIDC specification](https://solidproject.org/TR/oidc).
+
+Alternatively, for debugging purposes the server can be configured to always identify the client as a fixed WebID,
+or to allow the WebID to be set directly in the `Authorization` header.
+These can be configured by changing the `ldp/authentication` import in your configuration.
+
+## Authorization
+
+Two different authorization are supported for determining who has access to resources:
+
+* [Web Access Control](https://solidproject.org/TR/wac)
+* [Access Control Policy](https://solidproject.org/TR/acp)
+
+Or the server can be configured to not have any kind of authorization and allow full access to all resources.
+
+## Solid Protocol
+
+The [Solid Protocol](https://solidproject.org/TR/protocol) is supported, although it can be disabled if requested.
```

Comment:
I don't understand this sentence; what would happened if the Protocol was disabled? It's not just the LDPish interface, but also auth.

Reply:
Currently disabling one would also disable the other as there is only the LDP interface to access authorized data.

Reply:
Perhaps best to not mention that then?